src/nxdoc/web-ui/web-ui-upgrade-notes/web-ui-11-1-upgrade-notes.md

```diff
@@ -79,5 +85,81 @@ Removed variables can be safely deleted from your themes or elements. Deprecated
 * Removed `--nuxeo-results-view-min-height` (added in [NXP-27652](https://jira.nuxeo.com/browse/NXP-27652)).
 * Deprecated `--nuxeo-document-content-min-height` in favor of `--nuxeo-document-content-height` (affects `nuxeo-document-content`).
 * Deprecated `--nuxeo-document-trash-content-min-height` in favor of `--nuxeo-document-trash-content-height` (affects `nuxeo-document-trash-content`).
+* Deprecated `--nuxeo-document-creation-form-icon-width` and `--nuxeo-document-creation-form-icon-height` in favor of the mixin `--nuxeo-document-create-selection-icon` (affects `nuxeo-document-create`).
+
+### Nuxeo dropzone API change
+
+As of [NXP-28263](https://jira.nuxeo.com/browse/NXP-28263), `nuxeo-dropzone` exposes a new API that allows the element to be bound to a document field through the *value property*, which is more consistent with the API exposed by the other widgets. The previous API is deprecated but still supported (see [NXP-29391](https://jira.nuxeo.com/browse/NXP-29391)).
+
+### Use of atomic permissions
+
+Several elements of Nuxeo Web UI and Nuxeo Elements were updated to make use of `atomic permissions` instead of composite permissions. This was once one of the major causes to fork actions elements, which should now be reverted (see [WEBUI-5](https://jira.nuxeo.com/browse/WEBUI-5)).
+
+### Missing labels on actions menu
+
+Starting from 10.3, Nuxeo introduced an `actions menu` to wrap document actions (see [NXP-25146](https://jira.nuxeo.com/browse/NXP-25146)). Projects that forked action elements or have custom actions based on code from 9.10 won’t display a label unless they are upgraded as described in the documentation. This is mostly a cosmetic change, as actions will still work as they previously did, without the label.
+
+### Label deprecation
+
+Since [WEBUI-116](https://jira.nuxeo.com/browse/WEBUI-116), the `document history` and `audit page` use the same component and the new prefix `audit`. The following are some examples of the deprecated labels and their replacement:
+
+| Deprecated labels             | New Labels          |
+|-------------------------------|---------------------|
+| documentHistory.category      | audit.category      |
+| documentHistory.comment       | audit.comment       |
+| documentHistory.date          | audit.date          |
+| documentHistory.filter.after  | audit.filter.after  |
+| documentHistory.filter.before | audit.filter.before |
+
+### Reference to invalid packages in the project
+
+The reference to projects that no longer exist, such as `nuxeo-dam` or `nuxeo-spreadsheet`, or the presence of JSF specific contributions, might cause conflicts and prevent a project from working properly.
```

Comment:
```suggestion
The reference to packages that no longer exist.
```

Reply:
You're right about being `packages` instead of `projects`. In this case, we are talking about [this](https://github.com/nuxeo/nuxeo-web-ui/blob/master/.env.sample), as this is a dev oriented upgrade notes.
I've copied these from Gabriel notes, but I am assuming that the JSF specific contributions are related with the packages as well, but maybe @Gabez0r can elaborate a bit more.